### ISSUE TITLE
README states order of LngLat twice in the same way

### ISSUE THREAD

**Issue by JoshuaKahn** [davidcalhoun/tle.js#19]
In the README section describing the ground track code, it says:

```
// Returns points in [lng, lat] order when true, and [lng, lat] order when false.
  isLngLatFormat: true
```
However, the two orders are the same here ([lng, lat] and [lng, lat])

**Comment by davidcalhoun** [comment]
Thanks for the heads up!  Will fix this when I get a chance.  PRs are also accepted!

**Comment by davidcalhoun** [comment]
Fixed in v4.2.11
